packages/fast-usdc-contract/src/exos/advancer.ts

```diff
@@ -291,17 +297,13 @@ export const prepareAdvancerKit = (
          * @param {AdvancerVowCtx} ctx
          * @throws {never} WARNING: this function must not throw, because user funds are at risk
          */
-        onFulfilled(result, ctx) {
+        onFulfilled(result: undefined, ctx: AdvancerVowCtx) {
```

Comment:
Result is unused, maybe use `_`? I'm assuming this has to adhere to a specific interface, maybe we should cast the function to that interface explicitly? Also is it possible to remove the `@param` jsdoc stuff above? Maybe they're required if you want to include `@throws {never}`? Could that just be a regular comment instead?

Reply:
yeah it's the `@throws` that necessitates a jsdoc and once there is one the lint rule expects `@params` defined. I'm okay with it but if you want to change the lint config please file an issue or start a Slack thread